### PR TITLE
Use latest rules_go.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

**Comment by spxtr** [comment]
Woo, somehow I totally forgot about #2972 even though it was just a few days ago.

File: verify/verify-gofmt.sh [hunk 1]
```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env bash
```
Comment (ixdy) [inline verify/verify-gofmt.sh]: can we keep these but run them in travis or something?

Comment (spxtr) [inline verify/verify-gofmt.sh]: We could make a prow job that runs them I suppose.

**Comment by rmmh** [comment]
I'm fine with running verify scripts outside of bazel-- it's always been a hack!

**Comment by rmmh** [comment]
Can you move those verify checks to travis? Or add an issue for a separate job to run them?

/lgtm

**Comment by spxtr** [comment]
OK, verify-gofmt and verify-govet are running in their own jobs.

/test

**Pull request merged**